sandec/sandec.c

```diff
@@ -2,6 +2,7 @@
 #include <stdint.h>
 #include <stdio.h>
 #include <fcntl.h>
+#include <unistd.h>
```

Comment:
This is probably related to a bugfix in the compiler. Some of the calls in here, like `write`, are defined in `unistd.h`.